lib/insert-asset-manifest.js

```diff
@@ -0,0 +1,20 @@
+var mergeTrees = require('broccoli-merge-trees');
+var AssetManifestInserter = require('./asset-manifest-inserter');
+
+/**
+ * Given a tree, this function will insert the asset manifest into index.html.
+ *
+ * The `options` object is passed directly into the broccoli plugin.
+ *
+ * @public
+ * @param {Tree} tree
+ * @param {object} options
+ * @return {Tree}
+ */
+module.exports = function insertAssetManifest(tree, options) {
+  options = options || {};
+
+  var treeWithInsertedManifest = new AssetManifestInserter([tree], options);
```

Comment:
Correct me if my understanding is wrong, but this isn't actually the tree with an inserted manifest, correct? It's just the index files with the manifest inserted, right?

If so, can we rename this `indicesWithManifests` or something. Otherwise it makes the `mergeTrees` below feel weird.